test/library/library.go

```diff
@@ -0,0 +1,35 @@
+package library
+
+import (
+	"crypto/rand"
+	"fmt"
+	"math"
+	"math/big"
+	"path"
+	"regexp"
+	"runtime"
+	"testing"
+	"time"
+)
+
+var (
+	waitPollInterval = time.Second
```

Comment:
not sure this const makes sense. Poll intervals depend a lot on the actual thing you are polling, don't they?

Reply:
I guess as we add more tests we could stop using this constant if needed.

Reply:
Am against using consts before we need them. I know @mfojtik disagrees :)